logging/google/cloud/logging/logger.py

```diff
@@ -16,6 +16,10 @@
 
 from google.protobuf.json_format import MessageToDict
 from google.cloud._helpers import _datetime_to_rfc3339
+from google.cloud.logging.resource import Resource
+
+
+_GLOBAL_RESOURCE = Resource(type='global', labels={})
```

Comment:
I copied it from Monitoring and it's a namedtuple. I could make it not a named tuple or do Resource._from_dict({'type': 'global})